.gitignore

```diff
@@ -127,3 +127,9 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# helpers
+*helpers.py
```

Comment:
could you remove this?

Reply:
This has been removed and the file checked out of Git